app/wama.web/Views/AdminConsole/UserTool/UserAccountAddNewUser.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@{
+    @using WAMA.Core.Models.DTOs;
+
+    @model WAMA.Core.ViewModel.User.PatronUserAccountViewModel
+    string memberId = (Model?.MemberId) ?? ViewBag.MemberId;
+}
+
+<div class="col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2 col-xs-10 col-xs-offset-1">
+
+    @Html.Partial("DisplayError")
+    <form asp-action="UserAccountAddNewUser" asp-antiforgery="true" method="post">
```

Comment:
Hard-coded string for action, not best practice. Look at other forms (ViewAccount.cshtml, EditAccount.cshtml) I implemented on how I specify their action.